source/object-from-entries.ts

```diff
@@ -0,0 +1,36 @@
+/**
+A strongly-typed version of `Object.fromEntries()`.
+
+This is useful since `Object.fromEntries()` always returns `{ [k: string]: T }`. This function returns a strongly-typed object from given array of entries.
+
+- [TypeScript issues about this](https://github.com/microsoft/TypeScript/issues/35745)
+
+@example
+```
+import {objectFromEntries} from 'ts-extras';
+
+const stronglyTypedObjectFromEntries = objectFromEntries([
+	['a', 123],
+	['b', 'someString'],
+	['c', true],
+]);
+//=> {
+	a: number;
+	b: string;
+	c: boolean;
+}
```

Comment:
This will not render nicely in VS Code.

Reply:
Would updated version render nicely in VS Code?